changelog.d/3991.bugfix

```diff
@@ -0,0 +1 @@
+Fix bug when invalidating destination retry timings 
```

Comment:
can you update this to match the description of the original PR so that the changelog entries get combined, please.

Reply:
Oooh, good point